pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -188,6 +189,25 @@ func (k *Deployer) renderManifests(ctx context.Context, _ io.Writer, builds []bu
 		return nil, fmt.Errorf("reading config manifests: %w", err)
 	}
 
+	// A workaround for issue https://github.com/GoogleContainerTools/kpt/issues/1149
+	// fn-path cannot be recognized in kpt pipeline mode, and it results in that the kpt functions
+	// in are ignored.
```

Comment:
nit: Maybe expand this to explain what we're doing here for the workaround. I know you explained in the PR writeup but it might be nice to have it in the code as well